obs/qownnotes.spec

```diff
@@ -24,6 +24,13 @@ BuildRequires:  qt5-qtwebsockets-devel
 BuildRequires:  desktop-file-utils
 BuildRequires:  aspell-devel
 Requires:       qt5-qtsvg qt5-qtxmlpatterns qt5-qtwebsockets
+%endif
+
+%if 0%{?sle_version} == 150000 || 0%{?sle_version} == 150100 && !0%{?is_backports}
+BuildRequires:  libqt5-qtbase-devel libqt5-qtdeclarative-devel libQt5Svg-devel libQt5XmlPatterns-devel libqt5-qtwebsockets-devel
+#BuildRequires:  aspell-devel
+BuildRequires:  update-desktop-files
+Requires:       libQt5Svg5 libQt5Sql5 libQt5Sql5-sqlite libQt5Gui5 libQt5Network5 libQt5Widgets5 libQt5Xml5 libQt5XmlPatterns5 libQt5PrintSupport5 libQt5WebSockets5
 
 %else
```

Comment:
This else-clause will now also be execute for all other distributions other then SLE, don't you think? I'll try to put your new if-statement inside the old else-clause.

Reply:
so just

```
%if 0%{?sle_version} == 150000 || 0%{?sle_version} == 150100 && !0%{?is_backports}
#BuildRequires:  aspell-devel
%else
BuildRequires:  libaspell15
%endif
```

Reply:
maybe it's aspell-devel for both. I think libaspell15 is something different.. not really sure though

Reply:
we'll see, I manually added my lines in OBS, currently building

Reply:
SLE 15 is already built

Reply:
Looking good. Thanks for the if-statement!

Reply:
Great :tada: